tests/index.html

```diff
@@ -22,14 +22,14 @@
 
     {{content-for 'body'}}
     {{content-for 'test-body'}}
-    <script src="assets/vendor.js"></script>
-    <script src="assets/test-support.js"></script>
-    <script src="assets/dummy.js"></script>
-    <script src="assets/blanket-options.js"></script>
-    <script src="assets/blanket-loader.js"></script>
-    <script src="testem.js" integrity=""></script>
-    <script src="assets/tests.js"></script>
-    <script src="assets/test-loader.js"></script>
+    <script src="{{rootURL}}assets/vendor.js"></script>
+    <script src="{{rootURL}}assets/test-support.js"></script>
+    <script src="{{rootURL}}assets/dummy.js"></script>
+    <script src="{{rootURL}}assets/blanket-options.js"></script>
+    <script src="{{rootURL}}assets/blanket-loader.js"></script>
+    <script src="{{rootURL}}testem.js" integrity=""></script>
+    <script src="{{rootURL}}assets/tests.js"></script>
+    <script src="{{rootURL}}assets/test-loader.js"></script>
```

Comment:
I believe this line is no longer necessary and actually throws an error in the console when running tests because `ember-cli-test-loader` is now an npm dependency instead of a bower dependency.